lib/beaker/dsl/wrappers.rb

```diff
@@ -124,19 +131,43 @@ def powershell(command, args={})
           'NoProfile'       => '',
           'NonInteractive'  => ''
         }
+        encoded = false
         ps_opts.merge!(args)
         ps_args = []
         ps_opts.each do |k, v|
           if v.eql?('') or v.nil?
             ps_args << "-#{k}"
+          elsif k.eql?('EncodedCommand') && v
```

Comment:
If `k = EncodedCommand` and `v = nil`, then the result will be `ps_args = [ ..., -EncodedCommand]`. 

Is that command argument valid?